assets/node.yaml

```diff
@@ -16,8 +16,7 @@ spec:
       serviceAccount: aws-ebs-csi-driver-node-sa
       priorityClassName: system-node-critical
       tolerations:
-        - key: CriticalAddonsOnly
-          operator: Exists
+        - operator: Exists
```

Comment:
IIRC this is tolerating *any* taints regardless of the key.

IMO we only want to tolerate the `node-role.kubernetes.io/master` key:

```json
# Get one of my master nodes
oc get node/ip-10-0-130-8.us-east-2.compute.internal -o json | jq .spec.taints
[
  {
    "effect": "NoSchedule",
    "key": "node-role.kubernetes.io/master"
  }
]
```

As in:

```
      tolerations:
      - key: node-role.kubernetes.io/master
        effect: NoSchedule
```

Reply:
Tolerating everything is probably the best thing to do, because we don't know what node could want to run a pod with a PVC provided by the driver.

This is the same as image registry or SDN DaemonSets that tolerate everything.

Renamed to Tolerate any taints.

Reply:
ack